src/PhysicalOutbox.StorageTable.AcceptanceTests/When_no_partition_key_is_configured.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NServiceBus.AcceptanceTesting.Support;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_no_partition_key_is_configured : NServiceBusAcceptanceTest
```

Comment:
Both the test would also be true for the logical outbox part not just the physical right? So if we did move them into the appropriate shared folder and added a similar table client holder check right before https://github.com/Particular/NServiceBus.Persistence.AzureTable/blob/master/src/NServiceBus.Persistence.AzureTable/Outbox/LogicalOutboxBehavior.cs#L57-L58 those acceptance tests would cover both phases properly right?

Reply:
Yea, I’ll give it a try

Reply:
Moved both now, please take a look